lottie/src/main/java/com/airbnb/lottie/model/layer/BaseLayer.java

```diff
@@ -315,7 +315,22 @@ public void draw(Canvas canvas, Matrix parentMatrix, int parentAlpha) {
       }
 
       // Clear the off screen buffer. This is necessary for some phones.
-      clearCanvas(canvas);
+      if (getBlendMode() != LBlendMode.MULTIPLY) {
+        clearCanvas(canvas);
+      } else {
+        // Due to the difference between PorterDuffMode.MULTIPLY (which we use for compatibility
+        // with Android < Q) and BlendMode.MULTIPLY (which is the correct, alpha-blended mode),
+        // we will alpha-blend the contents of this layer on top of a white background before
+        // we multiply it with the opaque substrate below (with canvas.restore()).
+        //
+        // Since white is the identity color for multiplication, this will behave as if we
+        // had correctly performed an alpha-blended multiply (such as BlendMode.MULTIPLY), but
+        // will work pre-Q as well.
+        Paint solidWhite = new Paint();
```

Comment:
Could you create this as a nullable class field that you set if null? That way it gets created once and lazily? You can also use `LPaint()` instead which is mostly the same but does slightly fewer allocations for things that aren't needed.

Reply:
Sure thing!